docs/standard/linq/snippets/preserve-white-space-serializing/.gitignore

```diff
@@ -0,0 +1,2 @@
+bin/
```

Comment:
Let's delete this file. It should already be part of the .gitignore in the root of the repository.